ccd-claim-migration/src/main/java/uk/gov/hmcts/cmc/ccd/migration/services/ClaimMigrator.java

```diff
@@ -45,33 +62,213 @@ public void migrate() {
         AtomicInteger updatedClaims = new AtomicInteger(0);
         AtomicInteger failedMigrations = new AtomicInteger(0);
 
-        notMigratedClaims.forEach(claim -> {
-            try {
-                logger.info("\t\t start migrating claim: " + claim.getReferenceNumber());
+        notMigratedClaims.sort(Comparator.comparing(Claim::getId));
```

Comment:
We need to process claims in reverse order. `Comparator.comparing(Claim::getId).reversed()`

Reply:
done